### PR TITLE
added swisslandstats-geopy recipe meta.yaml

### PR DESCRIPTION
**Pull request by martibosch**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by martibosch** [comment]
The osx_64 checks seemingly fail because of [a ModuleNotFoundError: No module named 'conda_forge_ci_setup'](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=199350&view=logs&j=c572e5d5-7ba7-55ce-5575-7ce1d9f7161e&t=fe245b22-2f08-5a76-43b2-e02a18e3fefa&l=108) which clearly appears to be an issue of the conda-forge/Azure platform.

How should we proceed? I wonder whether re-triggering the builds might do.

File: recipes/swisslandstats-geopy/LICENSE.txt [hunk 1]
```diff
@@ -0,0 +1,674 @@
+                    GNU GENERAL PUBLIC LICENSE
```
Comment (synapticarbors) [inline recipes/swisslandstats-geopy/LICENSE.txt:1]: It looks like the license file is already in the source distribution so this copy can be removed.

Comment (martibosch) [reply recipes/swisslandstats-geopy/LICENSE.txt:1]: I have removed it from this repo in [f6cd132](https://github.com/conda-forge/staged-recipes/pull/12431/commits/f6cd132a13bb36e8794c1aa62b1c5bda79c9f9bf)

File: LICENSE.txt [hunk 2]
```diff
@@ -1,27 +0,0 @@
-Copyright (c) 2015-2018, conda-forge
```
Comment (synapticarbors) [inline LICENSE.txt:1]: Please do not delete this license. It is for conda-forge itself, not for the recipe you are trying to package. You need to delete the one in `recipes/swisslandstats-geopy/LICENSE.txt`.

Comment (martibosch) [reply LICENSE.txt]: Sorry, my mistake. I have amended it in [a7437d6](https://github.com/conda-forge/staged-recipes/pull/12431/commits/a7437d6760b732ec5beec93f47d1407dcb8505b0). Thank you for pointing it out.

**Pull request merged**